pages/index.tsx

```diff
@@ -23,24 +23,41 @@ import { CreateListProvider, useCreateList, usePagination } from 'hooks'
 import { Resource, RESOURCE_SEARCH_FIELDS } from 'models'
 import { NextPage } from 'next'
 import Head from 'next/head'
-import { useEffect, useMemo, useState } from 'react'
+import { useMemo, useState } from 'react'
 import { Search } from 'react-feather'
 import { useQuery } from 'react-query'
-import { fuseSearch, onEnter } from 'utils'
+import Select from 'react-select'
+import { fuseSearch } from 'utils'
 
 export const HomePage: NextPage = () => {
-  const { data: allResources } = useQuery<Resource[], Error>(
-    ['allResources'],
-    () => getAllResources()
+  const { data: allResources } = useQuery('getAllResources', () =>
+    getAllResources()
   )
 
-  const [filteredResources, setFilteredResources] = useState(allResources ?? [])
-  // Update resources state when query finishes
-  useEffect(() => {
-    if (allResources) {
-      setFilteredResources(allResources)
-    }
```

Comment:
Deleting because `filteredResources` is now memoized instead of stored in React state - I think this makes more sense since we get all the resources and filter them client-side so searching should be triggered on-change instead of via a button.